src/sphinx_tags/__init__.py

```diff
@@ -41,14 +56,21 @@ def run(self):
         if not (self.arguments or self.content):
             raise ExtensionError("No tags passed to 'tags' directive.")
 
-        tagline = []
+        page_tags = []
         # normalize white space and remove "\n"
         if self.arguments:
-            tagline.extend(self.arguments[0].split())
+            page_tags.extend(
+                [_normalize_tag(tag) for tag in self.arguments[0].split(",")]
```

Comment:
Using `_normalize_tag()` here (and below on line 69) strips out any special characters in the displayed tags:
![image](https://github.com/melissawm/sphinx-tags/assets/419936/0611ba02-614e-4764-abd2-0cb8e3694c27)

Changing that to `tag.strip()` works as expected:
![image](https://github.com/melissawm/sphinx-tags/assets/419936/b66d6562-e20c-46d0-81da-bd532f4a00b8)